framework/auth/views.py

```diff
@@ -79,6 +78,18 @@ def reset_password_get(auth, uid=None, token=None):
     }
 
 
+@block_bing_preview
+@collect_auth
+def reset_password_get(auth, uid=None, token=None):
+    return _reset_password_get(auth, uid=uid, token=token)
+
+
+@block_bing_preview
+@collect_auth
+def reset_password_institution_get(auth, uid=None, token=None):
+    return _reset_password_get(auth, uid=uid, token=token)
```

Comment:
There is no need to add an extra layer since the routes definition can handle it by mapping the path to the same view with a different mako. Here is the example:

```python
Rule(
    '/resetpassword-institution/<uid>/<token>/',
    'get',
    auth_views.reset_password_get,
    OsfWebRenderer('public/resetpassword_institution.mako', render_mako_string, trust=False)
),
```

Reply:
D'oh!  I did this in my original implementation, and it didn't work.  It turns out our [`process_rules` function](https://github.com/CenterForOpenScience/osf.io/blob/develop/framework/routing/__init__.py#L159-L169) maps the `Rule`s by building an identifier based off the `view` name and the `renderer` class (not including the args).  The calculated `endpoint` variable for the above `Rule` would be `OsfWebRenderer__forgot_password_get`.   When I used the same view & renderer and only changed the template, one always overwrote the other. That's why I had to break them out into wrapper methods that call the common code.  I thought I had docced this in the commit message, but alas, no.  Sorry about that!  I won't change the code above, but I will amend the commit message to include the reason.  Does that sound good?

Reply:
👍 Yes, and thanks for the explanation. I thought it might work but didn't try the change out locally.